app/views/employees/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<div class="user_info">
+    <h1><%= @employee.last_name %></h1>
+    <h1><%= @employee.first_name %></h1>
+    <h1><%= @employee.middle_name %></h1>
+</div>
+<div class="user_info">
+    <% unless @employee.position_histories.first.nil? %>
+        <h3>Position: <%= @employee.position_histories.last.position.name %></h3>
```

Comment:
would be good to have `def last_position` method in the decorator. In order to get rid of triggering db on the view side.